src/main/java/seedu/duke/commands/AddCheatSheetCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.duke.commands;
+
+import seedu.duke.editor.TextEditor;
+import seedu.duke.exception.CommandException;
+import seedu.duke.module.Module;
+import seedu.duke.module.ModuleList;
+import seedu.duke.ui.UI;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static seedu.duke.common.Constants.CHEATSHEET_DIR;
+import static seedu.duke.common.Constants.FOLDER_PATH;
+import static seedu.duke.common.Constants.PATH_DELIMITER;
+import static seedu.duke.common.Messages.MESSAGE_CHEATSHEET_ADDED;
+import static seedu.duke.common.Messages.MESSAGE_CHEAT_SHEET_ALREADY_EXISTS;
+
+public class AddCheatSheetCommand extends Command {
+    public static String fileName;
+
+    public AddCheatSheetCommand(String nameOfFile) {
+        fileName = nameOfFile;
+    }
+
+    @Override
+    public void execute(UI ui) throws CommandException {
+        Module module = ModuleList.getSelectedModule();
+        String directoryPath = getDirectoryPath(module);
+        Path path;
```

Comment:
Is there a reason why this is separated from line 33? (Same for the other files)